app/cms/utils/mdx.tsx

```diff
@@ -209,13 +255,21 @@ async function compileMdxCached({
     checkValue: checkCompiledValue,
     getFreshValue: async () => {
       const compiledPage = await compileMdx<MdxPage["frontmatter"]>(
-        `docs/${fileOrDirPath}`,
+        fileOrDirPath,
         files
       )
       if (compiledPage) {
         return {
           ...compiledPage,
           fileOrDirPath,
+          editLink: [
+            "https://github.com",
+            owner,
+            repo,
+            "blob",
+            branch,
+            entry,
+          ].join("/"),
```

Comment:
branch and entry always url safe, or is something like `encodeURIComponent` needed?